packages/netlify-cms-widget-markdown/src/MarkdownControl/plugins/SelectAll.js

```diff
@@ -0,0 +1,14 @@
+import isHotkey from 'is-hotkey';
+
+const SelectAll = () => ({
```

Comment:
System select-all functionality doesn't work well in our Slate editor, this forces a proper selection.